code/datums/station_traits/neutral_traits.dm

```diff
@@ -0,0 +1,22 @@
+/datum/station_trait/bananium_shipment
+	name = "Bananium Shipment"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 5
+	report_message = "Rumors has it that the clown planet has been sending support packages to clowns in this system"
+	trait_to_give = STATION_TRAIT_BANANIUM_SHIPMENTS
+
+/datum/station_trait/unnatural_atmosphere
+	name = "Unnatural atmospherical properties"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 5
+	show_in_report = TRUE
+	report_message = "System's local planet has irregular atmospherical properties"
+	trait_to_give = STATION_TRAIT_UNNATURAL_ATMOSPHERE
+
+/datum/station_trait/unique_ai
+	name = "Unique AI"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 5
+	show_in_report = TRUE
+	report_message = "For experimental purposes, this station AI might show divergence from default lawset. Do not meddle with this experiment."
```

Comment:
```suggestion
	report_message = "For experimental purposes, this station's AIs might show divergence from the default lawset(s). Do not meddle with this experiment."
```

Reply:
The (s) doesn't make sense here. The AI only has one default one on our servers.